js/main.js

```diff
@@ -178,3 +179,21 @@ $form.addEventListener('submit', function () {
   $form.reset();
 })
 ;
+
+$pictureList.addEventListener('click', function () {
+  if (event.target.matches('.picButton') === false) {
+    return;
+  }
+  var newSave = {};
+  var $newPic = document.querySelectorAll('.newPic');
+  for (var i = 0; i < $newPic.length; i++) {
+    if (event.target.getAttribute('data-pic') === $newPic[i].getAttribute('data-pic')) {
```

Comment:
Why is this necessary? Can't you just do `newSave.url = event.target.getAttribute('data-pic');` without a need for a loop?